.github/workflows/go.yaml

```diff
@@ -0,0 +1,49 @@
+name: Go
+
+on:
+  push:
+  pull_request:
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.15
+      - run: go version
+      - run: go test -race -v ./...
+  build:
```

Comment:
I don't think we need this step? Any reason for adding this?

Reply:
I think it good to test for bulding on many OS

Reply:
It does not make sense, though. If we want to test multiple OSes, we want 32-bit vs 64-bit system, though.

Reply:
I've removed it